io-ballerina/src/io/character/channel_string_io.bal

```diff
@@ -50,12 +50,12 @@ public function channelReadLines(ReadableChannel readableChannel) returns @taint
     }
 }
 
-# Read channel content as a stream of lines.
+# Read channel content as a stream of lines
```

Comment:
Same concern as above. How does it delimit the content to lines?

Reply:
Yes, it is obvious right?. We don't need to specify it here.